### DISCUSSION TITLE
How to navigate to another drive

### DISCUSSION THREAD

**Discussion by rp1231** [helix-editor/helix#6600] [Q&A] [answered]
Let's say that I've opened helix from the default location in windows powershell.
If I press space, it opens the file picker.
But if I search, it only shows the files in the C drive.
How do I navigate to the D drive from within the file picker?
Is this possible in Helix?
Or do I have to navigate to D drive in powershell  and then open Helix again?
Thanks

**Comment by CptPotato** [comment, marked as answer]
You should be able to change to a different drive using the `:cd` command, so `:cd D:\` in your case.

**Reply by rp1231** [reply]
Thanks
